.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "dropwizard-swagger"]
```

Comment:
Oh! Never mind, forgot that I didn't update .gitmodules - had to do something funky with it prior to, forgot it afterwards.

Sent from my iPhone

> On Apr 14, 2015, at 10:34 PM, Carlton Whitehead <notifications@github.com> wrote:
> 
> In .gitmodules:
> 
> > @@ -0,0 +1,3 @@
> > +[submodule "dropwizard-swagger"]
> Why did you add this as a submodule? The dependency declaration in the pom.xml should suffice
> 
> —
> Reply to this email directly or view it on GitHub.
>